FURPS+.md

Jenna-P opened the pull request.

Jenna-P committed:
Create FURPS+.md

simon-stentoft reviewed (changes requested):
Skal hjælpe producenten ved at have lavet en produktionsklar tegning*
Måske ændre til: Skal hjælpe producenten med at lave en produktionsklar tegning.

*Applikationen skal være let at benytte og støtte brugeren mest muligt ved at præsentere alle tilvalg overskueligt*
Ville selv sætte et 'skal' ind: benytte og skal støtte.

*Alle målangivelser på skitserne skal være korrekt detaljerede så de kan anvende til at producere ud fra*
Ville lave lidt om og skrive: Alle målangivelser på skitserne skal være korrekt detaljerede så de kan anvendes i produktionen.

Jenna-P commented:
> Skal hjælpe producenten ved at have lavet en produktionsklar tegning*
> Måske ændre til: Skal hjælpe producenten med at lave en produktionsklar tegning.
> 
> _Applikationen skal være let at benytte og støtte brugeren mest muligt ved at præsentere alle tilvalg overskueligt_
> Ville selv sætte et 'skal' ind: benytte og skal støtte.
> 
> _Alle målangivelser på skitserne skal være korrekt detaljerede så de kan anvende til at producere ud fra_
> Ville lave lidt om og skrive: Alle målangivelser på skitserne skal være korrekt detaljerede så de kan anvendes i produktionen.

det lyder fint ;) jeg vil rette det

Jenna-P committed:
Update FURPS+.md

TheProphet4000 committed:
Update FURPS+.md

helt ærligt, hader Dansk grammatik... Jeg kan ingengang finde helt ud af det xD

TheProphet4000 committed:
Merge pull request #40 from Jenna-P/update_FURPS

Update FURPS+.md
